src/Cache/FileCache.php

```diff
@@ -86,7 +88,11 @@ public function isFresh(string $path, int $timestamp)
     public function read(string $path)
     {
         $cachePath = $this->getCachePathFor($path);
-        $feature = unserialize(file_get_contents($cachePath));
+        $feature = unserialize(
+            file_get_contents($cachePath)
+                ?: throw new CacheException(sprintf('Cache could not be loaded from file "%s".', $path)),
+            ['allowed_classes' => true],
```

Comment:
(IDE) this is more explicit, although maybe we might want to define exactly which classes for security reasons?